apps/web-giddh/src/app/ledger/ledger.component.html

```diff
@@ -1124,20 +1485,32 @@ <h3 class="ledger-main-head">
                     <button [hidden]="!closingBalanceBeforeReconcile || showLoader" class="btn btn-primary btn-sm mr-0"
                         (click)="getReconciledBack();">{{commonLocaleData?.app_back}}
                     </button>
-                    <div *ngIf="closingBalanceBeforeReconcile && !showLoader" class="box balance-detail mr-t2 mr-b2 clearfix text-center">
+                    <div *ngIf="closingBalanceBeforeReconcile && !showLoader"
+                        class="box balance-detail mr-t2 mr-b2 clearfix text-center">
                         <div class="row">
                             <div class="col-md-6 col-sm-6 col-12 text-right">{{localeData?.balance_bank}}
-                                <h3 class="font-20 primary-color">{{reconcileClosingBalanceForBank?.amount}}
-                                    {{reconcileClosingBalanceForBank?.type}}
+                                <h3 class="font-20 primary-color">
+                                    <!-- {{reconcileClosingBalanceForBank?.amount}}
+                                        {{reconcileClosingBalanceForBank?.type}} -->
+
+                                    <amount-field [amount]="reconcileClosingBalanceForBank?.amount" [currencySymbol]="false" [currencyCode]="false"
+                                        [bankType]="reconcileClosingBalanceForBank?.type">
```

Comment:
remove bankType from input as show it after amount-field

Reply:
Removed